plugins/embed-optimizer/hooks.php

```diff
@@ -81,7 +81,7 @@ function embed_optimizer_filter_oembed_html( string $html ): string {
 			// by preventing links in the hidden iframe from receiving focus.
 			if ( $html_processor->has_class( 'wp-embedded-content' ) ) {
 				$style = $html_processor->get_attribute( 'style' );
-				if ( $style ) {
+				if ( is_string( $style ) ) {
```

Comment:
This case only happens if it had parsed `<div style>` where there is no attribute value. Unlikely, but it is a hardening.